UnitySamples/FBLeaderboard/Docs/FBLeaderboardTutorial.md

```diff
@@ -0,0 +1,301 @@
+            m_chilliConnect.PlayerAccounts.LogInUsingFacebook(fbAccessToken, (request, response) => OnChilliConnectFBLoggedIn(response), (request, error) => OnChilliConnectLoginFailed(error));
+        }
+        else
+        {
+            //Check if we have a stored anonymous ChilliConnect token to login with
+            if(PlayerPrefs.HasKey("CCId") == true && PlayerPrefs.HasKey("CCSecret") == true)
+            {
+                m_chilliConnect.PlayerAccounts.LogInUsingChilliConnect(PlayerPrefs.GetString("CCId"), PlayerPrefs.GetString("CCSecret"), (loginRequest) => OnChilliConnectAnonLoggedIn(), (loginRequest, error) => Debug.LogError(error.ErrorDescription));
+            }
+            else
+            {
+                //Create a new ChilliConnect account and allow the player to attach FB later.
+                var requestDesc = new CreatePlayerRequestDesc();
+                m_chilliConnect.PlayerAccounts.CreatePlayer(requestDesc, (request, response) => OnChilliConnectAccountCreated(response), (request, createError) => Debug.LogError(createError.ErrorDescription));
+            }
+        }
+    }
+    else
+    {
+        Debug.LogError("Failed to initialise FB SDK");
+    }
+}
+```
+### Logging into ChilliConnect using FB
+
+Creating a new ChilliConnect account and logging in using the ID and secret are covered in the initial setup tutorial and should be familiar to you. Logging into ChilliConnect with the Facebook token uses the following API:
+
+```c#
+string fbAccessToken = Facebook.Unity.AccessToken.CurrentAccessToken.TokenString;
+m_chilliConnect.PlayerAccounts.LogInUsingFacebook(fbAccessToken, (request, response) => OnChilliConnectFBLoggedIn(response), (request, error) => OnChilliConnectLoginFailed(error));
+```
+
+A handy feature is that the response contains the player's FB name:
+
+```c#
+private void OnChilliConnectFBLoggedIn(LogInUsingFacebookResponse response)
+{
+    Debug.Log("ChilliConnect logged in with FB account for " + response.FacebookName);
+}
+```
+
+### Linking FB to a ChilliConnect account
+
+When 'LogInUsingFacebook' is called (either after FB init or via user button press), ChilliConnect will attempt to find a player account associated with the given FB access token. If no account exists then the FB player is not yet associated with a ChilliConnect account and we link the current logged in FB account with the current local ChilliConnect account. Future attempts to login with the FB access token will return this account.
+
+```c#
+private void OnChilliConnectLoginFailed(LogInUsingFacebookError error)
+{
+    if(error.ErrorCode == LogInUsingFacebookError.Error.LoginNotFound)
+    {
+        LinkFacebookAccountRequestDesc requestDesc = new LinkFacebookAccountRequestDesc(m_fbAccessToken);
+        m_chilliConnect.PlayerAccounts.LinkFacebookAccount(requestDesc, (request, linkResponse) => OnChilliConnectLinked(linkResponse), (request, linkError) => Debug.LogError(linkError.ErrorDescription));
```

Comment:
I don't think that situation can occur as you need to give the app permission to use FB and therefore cannot login to FB for the app outside of the app. However for safety I'll handle this case